.gitignore

```diff
@@ -353,4 +353,11 @@ MigrationBackup/
 .vscode/
 
 # Docker data
-/data/
\ No newline at end of file
+/data/
+
+# db & settings
+
+*.realm
+*.lock
+*.json
+dataStore/
```

Comment:
If you're seeing these outside of `bin/Debug/net8.0`, you're probably running Refresh incorrectly.

We also have a `.json` file in use not relating to Realm or Bunkum/Refresh: https://github.com/LittleBigRefresh/Refresh/blob/main/global.json